IhorKremenskiy/HW2/books.js

```diff
@@ -0,0 +1,97 @@
+
+let bookName = prompt("Please enter a book name"); // Asking a user to enter a book name
+let bookAuthor = prompt("Please enter a book author"); // Asking a user to enter a book author
+let bookYear = prompt("Please enter a book publication year"); // Asking a user to enter a book publication year
+let bookPages = prompt("Please enter a book pages count"); // Asking a user to enter a book pages count
+
+
+
+// When all questions answered, the user should be asked to _confirm_ book save:
+// - Would you like to add "%bookName%" to the library?
+// If the user chooses "OK" he should be notified with the message:
+// - "%bookName%" by  %bookAuthor% successfully saved to the library!
+// (example: "The Adventures of Tom Sawyer" by Mark Twain successfully saved to the library!)
+
+
+// If the user chooses "Cancel" program should stop.
+
+if (confirm (`Would you like to add ${bookName} to the library?`)){
+    alert (`${bookName} by ${bookAuthor} successfully saved to the library`)
+
+} else close
+   // The *bookManager* function should _return_ an object with all data collected and some functions implemented, object structure should be like:
+// {
+//   name:"",
+//   author:"",
+//   plublication_year:"",
+//   pages_count:"",
+//   getBookSize:function(){},
+//   getPublicationCentury:function(){}
+
+// }
+let bookManager = function (){
+    return {
+        name: `${bookName}`,
+        author: `${bookAuthor}`,
+        publication_year: bookYear,
+        pages_count: bookPages,
+        getBookSize: function (){
+            if (this.pages_count < 101){
+                return 'S';
+            } else if (this.pages_count < 251 && this.pages_count >= 101) {
+                return 'M';
+            } else if (this.pages_count < 501 && this.pages_count >= 251) {
+                return 'L';
+            } else  {
+                return 'XL';
+            }
+
+
+        },
+        getPublicationCentury: function (){
```

Comment:
The function should return the only century number, nothing more.
Also, the function should be more flexible and return a century for any year,

4025 year - 41,
105 - 2 century,
2019 - 21 century

Too many ifs in this function, there are a couple options to make this function one-liner, but you need [Arithmetic Operators ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Assignment_Operators) you learned ONLY to achieve the goal